lib/engine/corporation.rb

```diff
@@ -255,6 +255,14 @@ def share_percent
       @second_share&.percent || presidents_percent / 2
     end
 
+    def border?
+      false
+    end
+
+    def sym
```

Comment:
what's this?

Reply:
See above.

Reply:
Again, the UI expects blockers to have a sym method.

Reply:
Nevermind. Removed.